### ISSUE TITLE
Sort package versions DESC in UI

### ISSUE THREAD

**Issue by pmig** [glasskube/glasskube#308]
**Is your feature request related to a problem? Please describe.**
Currently versions are sorted ascending in the UI version dropdown.

See:
![image](https://github.com/glasskube/glasskube/assets/3041752/2ade63e2-6c94-4e6e-b87e-a30708284a99)

**Describe the solution you'd like**
The latest version should be on top of the list and other releases should be sorted [semver](https://semver.org/) DESC.

**Additional context**
Currently blocked by #311

**Comment by hanshal101** [comment]
Hey, @pmig would like to work on this issue. Can you please guide me through the folder where UI code is mentioned? As far as I see, it's the website folder, where all the UI components are present..

**Comment by hanshal101** [comment]
Well I searched about this and got the folder. Here I can see that we need to modify this code :
https://github.com/glasskube/glasskube/blob/8d8559e0fc938ff54ea348efc02095579dc3d24b/internal/web/templates/components/pkg-install-modal-versions.html#L8-L12

The code to get the version in descending order should be:
```
{{ $reversed := reverse $idx.Versions }}
{{ range $reversed }}
<option value="{{ .Version }}" {{ if eq .Version $idx.LatestVersion }} selected {{ end }}>
  {{ .Version }} {{ if eq .Version $idx.LatestVersion }} (latest) {{ end }}
</option>
{{ end }}
```

**Comment by pmig** [comment]
Thanks for the proposal @christophenne can give feedback on your suggestion. As our versions are follow the [SemVer](https://semver.org/lang/de/) specification string sort might not be sufficient.

**Comment by christophenne** [comment]
Hi @hanshal101, thanks for bringing this up. 
The list of versions we display right now might not be sorted at all, it's just exactly whatever we get from the package repository, so reversing that order will not be sufficient. 
Since we want semver-aware ordering, we will probably need to include some library helping us with that – there is the related ticket #311 where we also need semver-related functionality. Therefore we will set this one "on hold" until we have come up with a solution with #311.

**Comment by hanshal101** [comment]
Ok will try working on them.

**Comment by kosmoz** [comment]
Quick fix: simply invert the list returned by the server.
